react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -39,21 +40,24 @@ class Game extends Component {
     const moves = this.getMoves(history);
     const status = winner ? `Winner: ${winner}` : `Next player: ${xIsNext ? 'X' : 'O'}`;
     return (
-      <div className={styles.game}>
-        <div className={styles.gameBoard}>
-          <Board
-            squares={squares}
-            onClick={this.handleClick}
-          />
+      <Fragment>
+        <TopBar />
```

Comment:
If you need to use this component in more than one view, you can add it in your routing section instead.